packages/react/src/hooks/useParticipantTracks.ts

```diff
@@ -0,0 +1,25 @@
+import * as React from 'react';
+import type { TrackReference } from '@livekit/components-core';
+import { participantTracksObservable } from '@livekit/components-core';
+import { useObservableState } from './internal';
+import type { Participant, Track } from 'livekit-client';
+import { useMaybeParticipantContext } from '../context';
+
+/**
+ * @public
+ */
+export function useParticipantTracks(
+  sources: Track.Source[],
+  participant?: Participant,
```

Comment:
I think accepting a participant.identity is closer to streaming use case and you don't have to get a participant first. The other way around, it is simple to get the identity if you already have a participant object.